doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2472 @@
+  ...            dtype=int) # offset = 1*itemsize, i.e. skip first element
+  array([2, 3])
+
+This also works for functions and other objects that **you** create. Just remember to include a docstring with your function using a string literal (``""" """`` or ``''' '''`` around your documentation).
+
+For example, if you create this function:
+
+::
+
+  def double(a):
+    '''Return a * 2'''
+    return a * 2
+
+You can run:
+
+::
+
+  double?
+
+Which will return:
+
+::
+
+  Signature: double(a)
+  Docstring: Return a * 2
+  File:      ~/Desktop/<ipython-input-23-b5adf20be596>
+  Type:      function
+
+You can reach another level of information by reading the source code of the object you're interested in. Using a double question mark (``??``) allows you to access the source code.
+
+For example, running:
+
+::
+
+  double??
+
+Will return 
+
+::
+
+  Signature: double(a)
+  Source:   
+  def double(a):
+      '''Return a * 2'''
+      return a * 2
+  File:      ~/Desktop/<ipython-input-23-b5adf20be596>
+  Type:      function
+
+If the object in question is compiled in a language other than Python, using ``??`` will return the same information as ``?``. You'll find this with a lot of built-in objects and types, for example:
+
+::
+
+  len?
+
+**Output:**
+
+::
+
+  Signature: len(obj, /)
+  Docstring: Return the number of items in a container.
+  Type:      builtin_function_or_method
+
+and
+
+::
+
+  len??
+
+**Output:**
+
+::
+
+  ​Signature: len(obj, /)
+  Docstring: Return the number of items in a container.
+  Type:      builtin_function_or_method
+
+have the same output because they were compiled in a programming language other than Python.
+
+
+
+Working with mathematical formulas
+----------------------------------
+
+Implementing mathematical formulas that work on arrays is one of the things that make NumPy so highly regarded in the scientific Python community. 
+
+For example, this is the mean square error formula (a central formula used in supervised machine learning models that deal with regression):
+
+.. image:: images/np_MSE_formula.png
+
+Implementing this formula is simple and straightforward in NumPy:
+
+.. image:: images/np_MSE_implementation.png
+
+What makes this work so well is that ``predictions`` and ``labels`` can contain one or a thousand values. They only need to be the same size. 
+
+You can visualize it this way:
+
+.. image:: images/np_mse_viz1.png
+
+In this example, both the predictions and labels vectors contain three values, meaning ``n`` has a value of three. After we carry out subtractions the values in the vector are squared. Then NumPy sums the values, and your result is the error value for that prediction and a score for the quality of the model.
+
+.. image:: images/np_mse_viz2.png
+
+.. image:: images/np_MSE_explanation2.png
+
+
+How to save and load NumPy objects
+----------------------------------
+
+::
+
+  np.save()
+  np.savez()
+  np.savetxt()
+  np.load()
+  np.loadtxt()
```

Comment:
I wonder whether  the following would render better?
```
.. autosummary:

    numpy.save
    numpy.savez
    numpy.savetxt
    numpy.load
    numpy.loadtxt
```

Reply:
I see some inconsistency in these sections. In the first one it's all listed on one line, in most other sections it's one function per line. I assume the latter is intended. In that case I think I agree, `autosummary` should turn all these into links while preserving the list layout. Worth a try.

Reply:
I changed this one and will be happy to change the rest if that's best

Reply:
`autosummary` will work for most but not all entries in these boxes. E.g., `?`, `??`, `.flatten` won't work. When it does work, I think it looks better. The question is if it's useful/needed. Because the end of the sections does already link to docstrings - example for ravel/flatten:

<img width="456" alt="image" src="https://user-images.githubusercontent.com/98330/71917186-c5f50c00-317f-11ea-881c-4c598cdd77cb.png">

So my first preference would be to get rid of the boxes. If you prefer to keep them, then I would suggest to change to `autosummary` everywhere that's possible, and keep as is in a few places.

Reply:
My thinking is that autosummary is awesome, but it makes things look more complex and slightly more daunting. I'm in favor of keeping this tutorial lighter for real beginners, which is why I included the familiar style of linked content at the end of each section, like a "read more"-type thing. As always, just let me know what your preference is!

Reply:
That makes sense. In that case I'd suggest either removing, or changing to this:

<img width="438" alt="image" src="https://user-images.githubusercontent.com/98330/71925403-42441b00-3191-11ea-9d73-f503e8a0af85.png">

That's much less compact, and avoids adding visually heavy gray boxes that look the same as code example boxes.

Reply:
this was done, resolving